pom.xml

```diff
@@ -2373,9 +2374,23 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>flink1.17</id>
+      <properties>
```

Comment:
https://github.com/apache/flink/blob/release-1.17/pom.xml#L144

Well need to make sure flink 1.17 is using avro 1.11 I believe

Reply:
Done.